ext/sqlite3/sqlite3.stub.php

```diff
@@ -68,6 +68,10 @@
 const SQLITE3_DETERMINISTIC = UNKNOWN;
 #endif
 
+class SQLite3Exception extends \Exception
```

Comment:
```suggestion
/**
 * @strict-properties
 */
class SQLite3Exception extends \Exception
```

Reply:
Prevents adding dynamic properties to objects of type SQLite3Exception. I consider this generally useful for any newly added classes.